com.ibm.wala.core/src/main/java/com/ibm/wala/util/scope/JUnitEntryPoints.java

```diff
@@ -87,24 +162,20 @@
         TypeName.findOrCreateClass(targetPackageAtom, targetSimpleClassAtom);
     final Atom targetMethodAtom = Atom.findOrCreateAsciiAtom(targetMethodName);
 
-    if (DEBUG) {
-      System.err.println(("finding entrypoint " + targetMethodAtom + " in " + targetType));
-    }
+    logger.finer("finding entrypoint " + targetMethodAtom + " in " + targetType);
```

Comment:
In other places you pass a lambda to the logger API.  Why not here?

Reply:
Ths one has a seemingly simple concatenation. Creating a lambda has a slight overhead (in fact, we have been studying this recently). Ideally, you would want to use the lambda version where the concentration outweighs the overhead.

In contrast, the statement at https://github.com/wala/WALA/pull/669/files#diff-6fe8230dbe8437b11645de9ccd666b8eR86 calls a `toString()` method, and I don't know how involved that method is. To be on the safe side, I used a lambda to avoid possibly unnecessary and possibly costly string concatenation.